pkg/api/kubegraph/edges.go

```diff
@@ -237,7 +237,7 @@ func AddHPAScaleRefEdges(g osgraph.Graph) {
 		switch groupVersionResource.GroupResource() {
 		case kapi.Resource("replicationcontrollers"):
 			syntheticNode = kubegraph.FindOrCreateSyntheticReplicationControllerNode(g, &kapi.ReplicationController{ObjectMeta: syntheticMeta})
-		case deployapi.Resource("deploymentconfigs"):
+		case deployapi.Resource("deploymentconfigs"), deployapi.LegacyResource("deploymentconfigs"):
```

Comment:
@sttts how about switching to https://github.com/openshift/origin/blob/master/pkg/deploy/api/register.go#L55